apps/src/templates/codeReviewGroups/CodeReviewGroup.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FontAwesome from '../FontAwesome';
+import color from '@cdo/apps/util/color';
+import StudentGroup from './StudentGroup';
+
+export default function CodeReviewGroup({
+  droppableId,
+  members,
+  name,
+  onNameUpdate,
+  onDelete
+}) {
+  const handleNameUpdate = event => {
+    onNameUpdate(droppableId, event.target.value);
+  };
+  return (
+    <div style={styles.groupContainer}>
+      <div style={styles.headerContainer}>
+        <input
+          value={name}
+          style={styles.nameInput}
+          onChange={handleNameUpdate}
+          placeholder={'Enter a group name'}
+        />
+        <button
+          style={styles.deleteButtonContainer}
+          onClick={() => onDelete(droppableId)}
+          type={'button'}
```

Comment:
Since we only have an icon, and no text on the button, it's good to add an `aria-label` attribute such as `aria-label="delete group"`. This will add a description that can be recognized by accessible technology.